packages/vite/src/node/idResolver.ts

```diff
@@ -66,7 +69,7 @@ export function createIdResolver(
                 {
                   root: config.root,
                   isProduction: config.isProduction,
-                  isBuild: config.command === 'build',
+                  isBuild,
```

Comment:
Shouldn't this be instead? Does it actually need to set `isBuild: false`? If so, why?
```js
isBuild: config.command === 'build',
optimizeDeps: false,
```

Reply:
The full bundle mode also is build, here only add a condition with `config.command === 'build' || experimental.fullBundleMode`. The production mode maybe could be removed.
Here need to ensure consistency with build.